custom/enikshay/management/commands/print_referral_report_indicators.py

```diff
@@ -0,0 +1,287 @@
+import json
```

Comment:
This is just a little script to make writing data source indicators less painful. This script was used to generate several of the indicators in `person_for_referral_report.json`, and can be used again if changed need to be made.